background.js

```diff
@@ -32,10 +108,10 @@ function wildcard_matches_any(patterns) {
 // Filter out utm_* query parameters
 var clean_utm = build_query_param_remover(function(p) { return p.startsWith("utm_") });
 browser.webRequest.onBeforeRequest.addListener(
-    clean_utm, {
-        urls: ["<all_urls>"],
-        types: ["main_frame"]
-    }, ["blocking"]
+    clean_utm_req,
+    {urls: ["<all_urls>"],
```

Comment:
please change the brace style to:
```javascript
addListener(
    clean_utm_req,
    {
        urls: [...],
        types: [...],
    },
    ["blocking"],
);
```

Reply:
correct! I will commit all linting separately.